multisense_ros/cfg/multisense.cfg

```diff
@@ -331,6 +336,26 @@ for feature_name, feature in SupportedFeatures.__members__.items():
             gen.add("ground_surface_spline_draw_resolution", double_t, 0, "Resolution to draw resulting B-Spline model with in RVIZ", 0.1, 0.01, 1.0)
         #endif
 
+        if feature == SupportedFeatures.APRILTAG:
```

Comment:
FWIW @QuentinTorg made some pretty major mods to this .cfg routine. We might want to land those first